### PR TITLE
Add navigation tabs

### PR DESCRIPTION
**Pull request by bodichka11**

### CODE REVIEW

**Comment by bodichka11** [comment]
https://trello.com/c/J9arGLOV/153-navigation-in-the-header

https://github.com/BinaryStudioAcademy/bsa-2023-leetwars/assets/113329862/c7606985-6544-4ba9-98cb-df9ff78a5c56

File: frontend/src/app/shared/components/header/header.component.sass [hunk 1]
```diff
@@ -59,3 +59,38 @@
...
+    border-bottom: none
+    color: $white
+
+:host ::ng-deep tabset li
+    padding: 0
+    color: $white
+
+:host ::ng-deep tabset li.nav-item.active a.nav-link.active
+  color: #325602 
+  border-bottom: 2px solid #325602
```
Comment (Limbo2332) [inline frontend/src/app/shared/components/header/header.component.sass]: use color variables

File: frontend/src/app/shared/components/header/header.component.sass [hunk 2]
```diff
@@ -59,3 +59,38 @@
...
+    line-height: 40px
+    color: $white
+    cursor: pointer
+    border: none
+    padding: 13px 13px
+    transition: background-color 0.2s ease
+    user-select: none
+    &:hover
+        background-color: transparent !important
+        color: #325602
```
Comment (Limbo2332) [inline frontend/src/app/shared/components/header/header.component.sass]: the same here

File: frontend/src/app/shared/components/header/header.component.sass [hunk 3]
```diff
@@ -59,3 +59,38 @@
...
+    &:focus
+      color: #325602
+      border-bottom: 2px solid #325602
```
Comment (Limbo2332) [inline frontend/src/app/shared/components/header/header.component.sass]: and here

File: frontend/src/app/shared/components/header/header.component.sass [hunk 4]
```diff
@@ -59,3 +59,38 @@
...
+        
+  
+:host ::ng-deep tabset li a.active
+    background-color: transparent !important
+    color: #325602
```
Comment (Limbo2332) [inline frontend/src/app/shared/components/header/header.component.sass]: and here

**Review by Limbo2332 — CHANGES_REQUESTED** [review]

**Review by Limbo2332 — APPROVED** [review]

File: frontend/src/app/core/services/tab.service.ts [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    private activeTabKey = 'activeTab';
+
+    getActiveTab(): string | null {
+        return localStorage.getItem(this.activeTabKey);
+    }
+
+    setActiveTab(tabRoute: string): void {
+        localStorage.setItem(this.activeTabKey, tabRoute);
+    }
+}
```
Comment (apofis96) [inline frontend/src/app/core/services/tab.service.ts]: Why do we need to save a tab in localStorage?
Cannot we assume active tab based on the current active route?

**Review by apofis96 — CHANGES_REQUESTED** [review]

File: frontend/src/app/core/services/tab.service.ts (hunk 5 above)
Comment (bodichka11) [reply frontend/src/app/core/services/tab.service.ts]: Done

**Review by tatianahutii — APPROVED** [review]

**Pull request merged**